Bottom: 917a007e716ab5d2993d3d94b2979d15567f7ae9
Top:    97a8047552a2494c6b24e2dc01bb0fdb894b5f2f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 17:05:42 -0500

Refresh of known_hosts

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 4cc67d20..b6d7f58e 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -17,7 +17,7 @@
 
 - name: Populate known_hosts secret
   copy:
-    content: '{{ known_hosts }}'
+    content: "{{ known_hosts }}\n"
     dest: '{{ secrets_hostdir }}/{{ secrets_known_hosts }}'
 
 - name: Checkout image repo
diff --git a/roles/buildbot-worker-ntel/tasks/main.yml b/roles/buildbot-worker-ntel/tasks/main.yml
index 32136e2f..f60ef127 100644
--- a/roles/buildbot-worker-ntel/tasks/main.yml
+++ b/roles/buildbot-worker-ntel/tasks/main.yml
@@ -57,7 +57,7 @@
 
 - name: Populate known_hosts secret
   copy:
-    content: '{{ known_hosts }}'
+    content: "{{ known_hosts }}\n"
     dest: '{{ secrets_hostdir }}/{{ secrets_known_hosts }}'
 
 - name: Checkout image repo
diff --git a/roles/buildbot-worker-ptxdist/tasks/main.yml b/roles/buildbot-worker-ptxdist/tasks/main.yml
index 91dec6ce..4d2c9ef0 100644
--- a/roles/buildbot-worker-ptxdist/tasks/main.yml
+++ b/roles/buildbot-worker-ptxdist/tasks/main.yml
@@ -55,7 +55,7 @@
 
 - name: Populate known_hosts secret
   copy:
-    content: '{{ known_hosts }}'
+    content: "{{ known_hosts }}\n"
     dest: '{{ secrets_hostdir }}/{{ secrets_known_hosts }}'
 
 - name: Checkout image repo
@@ -65,9 +65,9 @@
     dest: '{{ image_dir }}'
     force: yes
 
-- template:
-    src: pip.conf.j2
-    dest: '{{ image_dir }}/buildbot/.pip/pip.conf'
+# - template:
+#     src: pip.conf.j2
+#     dest: '{{ image_dir }}/buildbot/.pip/pip.conf'
 
 - template:
     src: pydistutils.cfg.j2
